.github/workflows/canary.yml

```diff
@@ -0,0 +1,75 @@
+name: Canary
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: Test (Ruby ${{ matrix.ruby }}, Gemfile ${{ matrix.gemfile }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ['3.1', '3.2', '3.3', 'head', 'jruby-9.4', 'jruby-head']
+        gemfile:
+          - gemfiles/postgresql/master.gemfile
+          - gemfiles/mysql2/master.gemfile
+          - gemfiles/sqlite3/master.gemfile
+          - gemfiles/trilogy/master.gemfile
```

Comment:
@albus522 Thank you to update ci.

It seems a job of jruby & trilogy is unnecessary on canary ci, because jruby doesn't support trilogy.

If exclude jruby & trilogy on canary, adding below settings.
```
        exclude:
          # jruby doesn't support trilogy
          - ruby: 'jruby-9.4'
            gemfile: gemfiles/trilogy/master.gemfile
          - ruby: 'jruby-head'
            gemfile: gemfiles/trilogy/master.gemfile
```